_core/smart/overview.md

```diff
@@ -13,31 +13,34 @@ Clarity is a smart contracting language for use with the Stacks 2.0 blockchain.
 
 ## Smart contracts
 
+Smart contracts encode and enforce the rules for data curation, and are often used to ensure that users can exchange valuable data in an programmatic and secure way. For example, a smart contract could allow creators to add a new tv show to a streaming service, but require that users first pay for a decryption key before viewing it. To do this, the smart contract could be written such that the movie creator had to publicly disclose the decryption key in order to receive payment. The movie creators would therefore be incentivized to work on tv show episodes that people want to watch, based on the viewers' upfront payments that are locked-in until the creator releases the episode and the decryption key.
```

Comment:
IMO, not the best example. A smart contract could not determine if they actually made the show or not. Who knows what the "decryption key" is used for? The smart contract could only know if the key is a key for anything - nothing further.

You could tweak this example if we want to stick to it. This could be a DAO, and the customers would have to vote that the movie producer did, in fact, deliver. However, there are lots of further issues than this.

Another option is that the customers are simply pre-purchasing, and they can then use signatures later on when streaming to prove they're a paid customer. However, that's also not a very interesting solution, because it doesn't provide any more value than a centralized service.

Reply:
Another problem is that it's not possible for the smart contract to then only give the decryption key to the payers - once it's on the blockchain, anyone can see it.

There are just a lot of complications around the idea of "use smart contracts to pay for digital goods"

Reply:
yeah, that makes sense. do you have a simpler example in mind? something digestible but also exciting =

Reply:
Is "data curation" a commonly used phrase here? Perhaps `Smart contracts encode and enforce the rules for modifying a particular set of data as shared among people and entities who don't necessarily trust each other.` is a better way to state this?

Reply:
Let's continue to iterate on the example but release this one for now